bindings/pyroot/cppyy/CPyCppyy/CMakeLists.txt

```diff
@@ -93,6 +98,7 @@ foreach(val RANGE ${how_many_pythons})
   target_include_directories(${libname}
      PRIVATE
         ${CMAKE_SOURCE_DIR}/core/foundation/inc   # needed for string_view backport
+        ${CMAKE_BINARY_DIR}/ginclude
```

Comment:
Is this only needed for Windows? It does not interfere in the other platforms?

Reply:
> Is this only needed for Windows? It does not interfere in the other platforms?

I don't think so, the PR build (once it works) should show us if it has side effects